src/Adapter/LegacyContext.php

```diff
@@ -53,16 +54,22 @@ class LegacyContext
     /** @var Tools */
     private $tools;
 
+    /** @var Tools */
```

Comment:
Wrong namespace

Reply:
no because it is the `Adapter\Tools` class not the legacy one